wikibase/queryService/RdfNamespaces-Wikidata.js

```diff
@@ -0,0 +1,102 @@
+var wikibase = window.wikibase || {};
```

Comment:
@SrdjanStevanetic can we have this file under the Europeana folder, and keep the original file in the wikibase folder?